src/index1.html

```diff
@@ -13,27 +13,18 @@ <h1 class="general-title"> LeaderBoard</h1>
     <div class="leaderboard-cont">
       <div class="title-btn-cont">
         <h4> Recent Scores</h4>
-        <button type="button"> Refresh!</button>
+        <button type="button" id="refresh-btn"> Refresh!</button>
```

Comment:
- [ ] From the requirements of this project, when the refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. In your app, the refresh button doesn't do any action. Please let's fix this and get closer to approval.

Reply:
If you check the refresh button hits the API when it calls the render function because its function its to call the getGames APImethod and it does print the user entries, so from that side its ok actually, please be more careful when checking.

Reply:
resolved